client/web/compose/src/components/ModuleFields/Editor/File.vue

```diff
@@ -127,6 +212,92 @@ export default {
         this.value = attachmentID
       }
     },
+    async initializeWebcam () {
+      await this.$nextTick()
+
+      this.video = this.$refs.video
+      this.canvas = this.$refs.canvas
+      this.captureButton = this.$refs.captureButton
+      this.uploadButton = this.$refs.uploadButton
+      this.closeButton = this.$refs.closeButton
+
+      this.startWebcam()
+    },
+
+    startWebcam () {
+      // Get access to the camera
+      navigator.mediaDevices.getUserMedia({ video: true })
+        .then(stream => {
+          this.stream = stream
+          this.video.srcObject = stream
+          this.video.style.display = 'block'
+          this.closeButton.style.display = 'inline-block'
```

Comment:
Pretty sure you won't need any button refs if you bind it to v-ifs. Since what you're doing is exactly what v-ifs are for
You need to disable the capture button if the webcam is initializing.